src/oop_ext/interface/_interface.py

```diff
@@ -901,7 +973,7 @@ def _CalculateMro(class_, mro):
     return mro
 
 
-def _GetClassImplementedInterfaces(class_):
+def _GetClassImplementedInterfaces(class_: Type) -> FrozenSet[Type[Interface]]:
     cache = __ImplementedInterfacesCache
     result = cache.GetResult(class_)
     if result is not None:
```

Comment:
result here will return `Any` istead of `FrozenSet[Type[Interface]]`

Reply:
Again not seeing this error.

Reply:
Sure my bad, as mentioned [in this comment](https://github.com/ESSS/oop-ext/pull/32#discussion_r577797275) I activated an extra option in mypy believing that we should be a little more strict while using Any in a lib that expects its interfaces to be generic.

So in this original comment, I should have written the following message:

Running mypy with `--warn-return-any` (which checks if the function returns Any when the function informs a type) I noticed this error, which makes sense and perhaps needs more attention.

Reply:
Yeah it complains about any `Any` anywhere in the signature.

Anyway I added the appropriate comments at the top of the modules and updated the code accordingly. 👍